docs/schema-loader.md

```diff
@@ -714,3 +722,35 @@ SchemaLoader.repairTables(properties, serializedSchemaJson, tableReparationOptio
 // Delete tables.
 SchemaLoader.unload(properties, serializedSchemaJson, deleteCoordinatorTables);
 ```
+
+### Import tables
+
+You can import an existing JDBC database table to ScalarDB in a similar manner as the above sample. Note that the schema must be import-specific. See also the following document for the details.
+
+- [Importing existing tables to ScalarDB using ScalarDB Schema Loader](./schema-loader-import.md)
+
+{% capture notice--warning %} Attention
+
+You should carefully plan to import a table to ScalarDB in production because it will add transaction metadata columns to your database tables and the ScalarDB metadata tables. There would also be several differences between your database and ScalarDB and limitations.
+
+{{ notice--warning | markdownify }}
+
+An import sample is as follows:
+
+```java
+public class SchemaLoaderSample {
+  public static int main(String... args) throws SchemaLoaderException {
+    Path configFilePath = Paths.get("database.properties");
+    // "import_sample_schema.json" can be found in the "/sample" directory.
+    Path schemaFilePath = Paths.get("import_sample_schema.json");
+    Map<String, String> tableCreationOptions = new HashMap<>();
+
+    // Import tables.
+    // You can also use a Properties object instead of configFilePath and a serialized-schema JSON
+    // string instead of schemaFilePath.
+    SchemaLoader.load(configFilePath, schemaFilePath, tableCreationOptions, createCoordinatorTables);
```

Comment:
`importTables`, right?
```suggestion
    SchemaLoader.importTables(configFilePath, schemaFilePath, tableCreationOptions);
```

Reply:
Thank you! Fixed in e3dd00d.

Reply:
One more thing, it looks like the `importTables` method doesn't receive the `createCoordinatorTables` argument.

And maybe `tableCreationOptions` should be renamed to something like `tableImportOptions` option?

Reply:
Oops, sorry about that. I fixed it throughout the sample in 148d9e2. PTAL!